### DISCUSSION TITLE
AuthApiError: Invalid login credentials

### DISCUSSION THREAD

**Discussion by Overseer1** [supabase/supabase#19020] [Questions]
I'm getting this error when I'm trying to log in. The email and password are correct but it's saying that I'm inputting invalid credentials. And yes, I already used the onAuthStateChange and it shows that the event is in INITIAL_SESSION.
![image](https://github.com/supabase/supabase/assets/120192190/8ce3ec2a-6a56-4c2b-a6d4-8d27a57e2e0d)

**Comment by GaryAustin1** [comment]
I believe it returns that message if email and password do not match an existing users email and password for any reason.

**Reply by Overseer1** [reply]
but the thing is that the login credentials that I inputted were correct, it's just returning the AuthApiError message for some reason.

**Comment by SaddamMohsen** [comment]
did configure the supabase_api and url correctly and initialized supabase client
